pkg/metronome/types.go

```diff
@@ -119,6 +119,16 @@ type Schedule struct {
 	NextRunAt               string `json:"nextRunAt"`
 }
 
+// Queue contains all queued runs of a Job.
+type Queue struct {
+	JobID string `json:"jobId"`
+	Runs  []queuedRun `json:"runs"`
+}
+
+type queuedRun struct {
```

Comment:
What's the purpose of having a struct just to contain a string? Wouldn't `runIds  []string` in `Queue` be enough?